packages/forms/src/UIForm/fields/Date/DateTime/DateTime.component.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import memoizeOne from 'memoize-one';
+import InputDateTimePicker from '@talend/react-components/lib/DateTimePickers';
+import FieldTemplate from '../../FieldTemplate';
+import { isoDateTimeRegExp } from '../../../customFormats';
+import { WidgetUnhandleTypeError, WidgetUnexpectedTypeError } from '../WrongTypeError';
+import { generateDescriptionId, generateErrorId } from '../../../Message/generateId';
+
+const HANDLE_CONVERTION_TYPE = ['string', 'number'];
+const UNIQUE_ERROR_MESSAGE = 'The date is invalid. Expected format: YYYY-MM-DD HH:mm';
+const INVALID_DATE = new Date('');
```

Comment:
not a big fan to have invalid objects, it is harder to grab the edge case from the caller, why not throw or null ?